File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/pattern3/p9.c
#include<stdio.h>
void main(){
	
	int rows,num;
	char ch;
	printf("Enter num of rows:");
	scanf("%d",&rows);

	printf("Enter any num:");
        scanf("%d",&num);

	printf("Enter any character:");
        scanf(" %c",&ch);

	for(int i=1;i<=rows;i++){
		for(int j=1;j<=rows;j++){
			if(i%2 !=0){
				printf("%d\t",num);
			}else{
				printf("%c\t",ch);
			}
			num--;
			ch--;
		}
		printf("\n");
	}
}

File: java-main/java-main/Nestedforloop/p9.java
/* A 1 B 2
   A 1 B 2 
   A 1 B 2
   A 1 B 2 */

class Demo{

	public static void main(String[]args){

		for(int i=1;i<=4;i++){
			char ch='A';
			int num=1;
			for(int j=1;j<=4;j++){
				if(j%2==1){
					System.out.print(ch++ + " ");
				}else{
					System.out.print(num++ + " ");
				}
			}
			System.out.println();
		}
	}
}
